src/Makefile.am

```diff
@@ -42,8 +42,8 @@ endif
 EXTRA_DIST = \
 	bdftogd \
 	demoin.png \
-	entities.html \
-	entities.tcl \
+	entities.json \
```

Comment:
instead of checking it into git (since it's like 150KB), how about we have entities.py fetch it on the fly & cache it ?

so if src/entities.json doesn't exist, download https://html.spec.whatwg.org/entities.json and save it there.  then read it from that path.

you'll also want to update .gitignore.

there is somewhat an argument to be made about not being able to recreate this file reliably depending on the git version, but i'm not sure we really care about that when we're committing the generated header output to the repo.

wdyt ?

```
import urllib.request
...
if not os.path.exists("entities.json"):
  with open("entities.json", "wb") as fp:
    fp.write(urllib.request.urlopen("https://...").read())
```

Reply:
One concern I have here is if they update the json entities to have 3 unicode, or more, libgd would need modified to support that. It seems all the current ones are 1 or 2 for now.
I like the idea of keeping the json file on git, then if a dev decides to delete it and let the script snag a new one they can. I dunno how often they add to the spec, but most fonts i've tried are missing half the glyphs anyways.

Reply:
i don't think we have to worry about this too much.  we'll have the generated source files checked in, so releases would never run the script.  we aren't guaranteeing that every release will be able to generate the latest entities.json, or even that it even needs to run at all.